subsys/zigbee/osif/zb_nrf_transceiver.c

```diff
@@ -137,15 +136,24 @@ void zb_trans_start_get_rssi(zb_uint8_t scan_duration_bi)
 }
 
 /* Waiting for the end of energy detection procedure and reads the RSSI */
+#ifdef CONFIG_ZBOSS_LIBRARY_SOURCES
```

Comment:
Wha do you think about updating development libraries and changing this condition to the `CONFIG_ZIGBEE_LIBRARY_DEVELOPMENT`?

Reply:
I agree, this `#ifdef` woud have to be updated after update of development libraries anywany.
And when building ZBOSS from sources `CONFIG_ZIGBEE_LIBRARY_DEVELOPMENT` becomes a default and is enabled so it's safe.
I'll change it to  `CONFIG_ZIGBEE_LIBRARY_DEVELOPMENT` once the development ZBOSS is updated.